src/less/filter-button/base/filter-button.less

```diff
@@ -1,3 +1,7 @@
+div.filter-button__wrapper {
+    .filter-button-wrapper;
```

Comment:
Why use a mixin here?

Reply:
I believe I did this so that when I go to create the filter-menu-button it will be easier to drop in the mixins.